keps/sig-api-machinery/1929-built-in-default/README.md

```diff
@@ -276,12 +276,9 @@ endpoint today, but we have to decide what to do with the new default
 values of built-in types. The kube-openapi generated schemas will
 include them.
 
-We proppose to:
-1. leave CRDs not publishing defaults
-2. filter out defaults in the built-in type schemas too in a first step
-3. but possibly reconsider this behaviour in the future when the
-necessity of defaults for client-side merging (e.g. through kustomize)
-and influences on the ecosystem of default mis-use is better understood.
+We propose to:
```

Comment:
You should update the paragraph before since I don't think it's accurate.

Reply:
Done

Reply:
Done